backend/api/Areas/Admin/Mapping/Parcel/AddressMap.cs

```diff
@@ -0,0 +1,38 @@
+using Mapster;
```

Comment:
Instead of `Profile` we now use `IRegister` to identify classes that will provide the mapping configuration.